src/borg/manifest.py

```diff
@@ -148,7 +148,7 @@ class Manifest:
     class Operation(enum.Enum):
         # The comments here only roughly describe the scope of each feature. In the end, additions need to be
         # based on potential problems older clients could produce when accessing newer repositories and the
-        # tradeofs of locking version out or still allowing access. As all older versions and their exact
+        # trade offs of locking version out or still allowing access. As all older versions and their exact
```

Comment:
https://dict.leo.org/german-english/trade-off they say it is either `tradeoff` or `trade-off` (but not `trade off`).

Reply:
I should have checked this one more closely. It didn't seem quite right to my mind, but it's too early in my morning.

Reply:
ngrams says that `trade-offs` is winning:
https://books.google.com/ngrams/graph?content=tradeoffs%2Ctrade-offs&year_start=1800&year_end=2019&corpus=26&smoothing=3

https://trends.google.com/trends/explore?q=tradeoffs,trade-offs

Looks like the dashed form is preferred